example-config.json

```diff
@@ -38,5 +38,15 @@
       "nrBlocks": 10,
       "blockByteSize": 100
     }
+  ],
+  "demo": [
```

Comment:
Call it `readDrive` so it's clear what the experiment is about

Reply:
i've used `readHyperdrive` for the sake of consistency.